test/simulation/xud_test.go

```diff
@@ -409,18 +459,25 @@ func launchNetwork(noBalanceChecks bool) (*xudtest.NetworkHarness, func()) {
 		} else {
 			log.Printf("xud network harness teared down")
 		}
+		log.Printf("cleaning up processes...")
+		cleanupCmd := exec.Command("./cleanup-processes.sh")
```

Comment:
Is this here basically to ensure that all the processes are actually killed? I know sometimes when the tests crash currently the processes are left running, but on a clean run all processes are stopped for me. I'm thinking that might still be the case if we don't reach this code, we could have some processes that don't get killed.

Reply:
This is to ensure we also kill geth and raiden after a successful test run (managing and keeping track of the process ids was troublesome). It looks like if the test suite fails this script won't be executed. 

This script can also be used manually for cleaning up while debugging the test suite.